tests/cases/conformance/types/mapped/mappedTypeErrors.ts

```diff
@@ -0,0 +1,62 @@
+// @strictNullChecks: true
+// @declaration: true
+
+type Partial<T> = {

```

Comment:
since these will soon be part of lib.d.ts, probably the test versions should have mangled names to avoid duplicate identifiers.